src/emulator/functionsEmulator.ts

```diff
@@ -818,7 +832,127 @@ export class FunctionsEmulator implements EmulatorInstance {
     return process.execPath;
   }
 
-  invokeRuntime(frb: FunctionsRuntimeBundle, opts: InvokeRuntimeOpts): RuntimeWorker {
+  getUserEnvs(): Record<string, string> {
+    const projectInfo = {
+      functionsSource: this.args.functionsDir,
+      projectId: "local", // "local" is reserved for Emulator use in functions env var.
```

Comment:
Can you elaborate on this?

Reply:
We decided to reserved `.env.local` file for setting env vars for the Functions Emulator (go/cf3-env-vars-dotenv).

Now that you point this out, I feel like this logic is better kept interrnal in the functoinsEnv module. Refactored to make finding ".env.local" a concern of the functions env var module.